lib/errorHandlers/index.ts

```diff
@@ -7,37 +6,39 @@ const {
   isValidationError,
   isMissingScopeError,
 } = require('@hubspot/local-dev-lib/errors/index');
+import { FileSystemError } from '@hubspot/local-dev-lib/models/FileSystemError';
+import { HubSpotHttpError } from '@hubspot/local-dev-lib/models/HubSpotHttpError';
+import { BaseError } from '@hubspot/local-dev-lib/types/Error';
 const { shouldSuppressError } = require('./suppressError');
 const { i18n } = require('../lang');
 const util = require('util');
-const { isAxiosError } = require('axios');
 
 const i18nKey = 'lib.errorHandlers.index';
 
-function logError(error, context = {}) {
+export function logError(error: unknown, context?: typeof ApiErrorContext) {
   debugError(error, context);
 
   if (
     shouldSuppressError(error, context) ||
-    shouldSuppressError(error, error.context)
+    shouldSuppressError(error, (error as HubSpotHttpError).context)
```

Comment:
If we only want to call this for `HubSpotHttpError` we could update this to `isHubSpotHttpError(error) && shouldSuppressError(error, error.context)`, of if we want to call it in all cases we can use `shouldSuppressError(error, 'context' in error ? error.context: undefined)` so we don't make assumptions about the errors type.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1251/commits/9fc06a16fccfda6b8e5d060b6c4703e0a83a0d07